modules/flowable-cmmn-rest/src/main/java/org/flowable/cmmn/rest/service/api/runtime/caze/CaseInstanceIdentityLinkResource.java

```diff
@@ -54,9 +54,15 @@ public RestIdentityLink getIdentityLink(@ApiParam(name = "caseInstanceId") @Path
             @ApiParam(name = "type") @PathVariable("type") String type,
             HttpServletRequest request) {
 
-        CaseInstance caseInstance = getCaseInstanceFromRequest(caseInstanceId);
+        CaseInstance caseInstance = runtimeService.createCaseInstanceQuery().caseInstanceId(caseInstanceId).singleResult();
+        if (caseInstance == null) {
+            throw new FlowableObjectNotFoundException("Could not find a case instance with id '" + caseInstanceId + "'.");
+        }
 
         validateIdentityLinkArguments(identityId, type);
+        if (restApiInterceptor != null) {
+            restApiInterceptor.accessCaseInstanceIdentityLinks(caseInstance);
```

Comment:
This can be more specific, since we are looking for a specific identity link. Lets move this to something like `accessCaseInstanceIdentityLink(caseInstance, link)` just before we return.

Reply:
done